rgb2hex.js

```diff
@@ -41,9 +41,9 @@
         alpha = parseFloat(Math.round(alpha * 100), 10) / 100;
 
         return {
-            hex: digits[1] + '#' + rgb.toString(16),
```

Comment:
I hope my removal of this is fine. I can't imagine a case where this would be wanted, and removing it makes the fix cleaner.